### PR TITLE
json-login-ldap

### PR DESCRIPTION
**Pull request by ronfroy**
https://github.com/symfony/symfony/pull/27650

### CODE REVIEW

File: reference/configuration/security.rst [hunk 1]
```diff
@@ -164,7 +164,7 @@ logout_on_user_change
 **type**: ``boolean`` **default**: ``false``
 
 .. versionadded:: 3.4
-    The ``logout_on_user_change`` option was introduced in Symfony 3.4.
+The ``logout_on_user_change`` option was introduced in Symfony 3.4.
```
Comment (nicolas-grekas) [inline reference/configuration/security.rst]: should be reverted

**Comment by ronfroy** [comment]
@nicolas-grekas done

File: reference/configuration/security.rst [hunk 2]
```diff
@@ -191,8 +191,8 @@ Authentication
 ~~~~~~~~~~~~~~
 
 You can authenticate to an LDAP server using the LDAP variants of the
-``form_login`` and ``http_basic`` authentication providers. Simply use
-``form_login_ldap`` and ``http_basic_ldap``, which will attempt to
+``form_login``, ``http_basic`` and ``json_login`` authentication providers. Simply use
```
Comment (HeahDude) [inline reference/configuration/security.rst]: This should be reverted and back ported in 3.4

Comment (javiereguiluz) [reply reference/configuration/security.rst]: I don't understand this comment. The entire pull request should be done in 3.4 branch? Only part of the changes? If so, which ones? Thanks!

Comment (HeahDude) [reply reference/configuration/security.rst]: Only the change on this line, adding `json_login`, it is a Symfony 3 feature.

Comment (ronfroy) [reply reference/configuration/security.rst]: @HeahDude it's related to LDAP, so i think it's not related to SF3

Comment (HeahDude) [reply reference/configuration/security.rst]: Ldap added its own very lately. See https://github.com/symfony/symfony/pull/18952

Comment (ronfroy) [reply reference/configuration/security.rst]: @HeahDude 
yes, but the LDAP variant of json_login does not exist on Symfony 3

Comment (HeahDude) [reply reference/configuration/security.rst]: Yes, and the change on this line has nothing to do with it :). There are two different changes in this PR, one must be back ported to previous version.

Comment (ronfroy) [reply reference/configuration/security.rst]: ```You can authenticate to an LDAP server using the LDAP variants of the``` it has to do with it.  On SF 3 there are no variant of json_login so json_login is not mention as a variant.

Comment (HeahDude) [reply reference/configuration/security.rst]: Yes you're right, sorry for the confusion.

**Review by HeahDude — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@ronfroy thank you for having contributed this feature and its docs! Congrats on your first Symfony Docs contribution!

**Pull request merged**